utilities/copyoffload_migration.py

```
@@ -407,6 +407,97 @@ def _find_populate_pods(ocp_admin_client: DynamicClient, namespace: str, migrati
+                captured_logs.append(pod_data)
+                LOGGER.debug(f"Captured logs from populate pod '{pod.name}' (phase: {pod_data['phase']})")
+            except Exception as pod_err:
+                LOGGER.warning(f"Failed to capture logs from pod '{pod.name}': {pod_err}")
+
+        if captured_logs:
+            fixture_store["populate_pod_logs"][migration_uid] = captured_logs
+            LOGGER.info(
+                f"Successfully captured {len(captured_logs)} populate pod log(s) for migration '{migration_uid}'"
+            )
+
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)